src/02b-pandas.ipynb

```diff
@@ -0,0 +1,1871 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "cbb2bd08",
+   "metadata": {},
+   "source": [
+    "# Pandas introduction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a710402b-3b26-41bf-9738-c19522c51c41",
+   "metadata": {},
+   "source": [
+    "# 0. Introduction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "546e1e09",
+   "metadata": {},
+   "source": [
+    "## 0.1. Why Pandas?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "00315f42",
+   "metadata": {},
+   "source": [
+    "Pandas is a Python package that has become a standard for data science. It is very relevant because:\n",
+    "+ It allows the use of data structures similar to SQL tables or spreadsheets.\n",
+    "+ It includes several functions that make it easy to work with data sets.\n",
+    "+ It is also possible to create graphics!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3d01ed76",
+   "metadata": {},
+   "source": [
+    "Pandas documentation:\n",
+    "+ [Documentation](https://pandas.pydata.org/docs/)\n",
+    "+ [Pandas Cheat Sheet](https://pandas.pydata.org/Pandas_Cheat_Sheet.pdf)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f40ff572-db79-4420-8384-73fcfcaccce6",
+   "metadata": {},
+   "source": [
+    "Some equivalances between Pandas and SQL can be found in the [Pandas documentation](https://pandas.pydata.org/docs/getting_started/comparison/comparison_with_sql.html)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "be2bcbd8",
+   "metadata": {},
+   "source": [
+    "## 0.2. Import the package"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fdcb84da",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-warning\">\n",
+    "    Firstly, you have to install the package using the Anaconda Prompt:\n",
+    "    <br>\n",
+    "    $ pip install pandas\n",
```

Comment:
Perhaps we could change this to `conda install pandas`? Or at least, explain to people that they can install things in two ways, either using `conda` or using `pip`. Some packages are more difficult to install using `pip` while they are functioning easier using `conda`.

Reply:
Yes, it is perfect for me to include both options and explain `conda` before `pip`.

Reply:
OK, I've added a small explanation.